packages/espressocash_backend/pubspec.yaml

```diff
@@ -19,7 +19,8 @@ dependencies:
   shelf: ^1.4.0
   shelf_router: ^1.1.3
   shelf_static: ^1.1.1
-  solana: ^0.26.1
+  solana:
+    path: ../solana
```

Comment:
It doesn't work that way, for package to be uploaded to Google Cloud, the dependency should come from `pub.dev`.